astroplan/plots/tests/test_sky.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import pytest
+
+try:
+    import matplotlib
+    HAS_MATPLOTLIB = True
+except ImportError:
+    HAS_MATPLOTLIB = False
+
+
+# TODO: replace this with actual plot checks once these
+# issues are resolved:
+# https://github.com/astropy/astroplan/issues/65
+# https://github.com/astropy/astroplan/issues/74
+@pytest.mark.skipif('not HAS_MATPLOTLIB')
+@pytest.mark.mpl_image_compare
```

Comment:
Wouldn't you need to give this an image name? Or this just works as there is only one image tested at the moment? (I just wonder as I'm trying to do this for photutils at the moment.)